frontend/views/containers/group-settings/GroupRulesSettings.vue

```diff
@@ -58,6 +71,12 @@ export default ({
       // because the group disappears. it's not a big deal though
       return this.groupProposalSettings() || {}
     },
+    votingRulesSorted () {
+      // NOTE: temporarily hiding DISAGREEMENT_RULE from the settings.
+      // TODO: once disagreement-rule implementation is ready, put this back to
+      //       return this.proposalSettings.rule === RULE_DISAGREEMENT ? [RULE_DISAGREEMENT, RULE_PERCENTAGE] : [RULE_PERCENTAGE, RULE_DISAGREEMENT]
+      return [RULE_PERCENTAGE]
+    },
```

Comment:
@taoeffect just reverted the code back to what they were and hided the 'disagreement-rule' by making this change here with the comment explaining the context. Thought this is the way that requires the least amount of code change, hence making things easier when we want to re-enable it in the future. but pls let me know if you have other thoughts.
cheers,

Reply:
@SebinSong Thanks - however this is insufficient to hide the setting from the user.

Now, users will just be confused as to what's going on. There's a setting that's broken completely, they can't even switch to it.

When I said "hide" and "comment out" - I meant it. :P

Please keep this old code around, but the user must not see this alternative voting system until we get it working.

Reply:
Oops, so ... again it seems like the first time I reviewed this I didn't get the latest updates some how. Now it is working as expected.

Something odd going on with my `git pr-id` script. Ignore what I wrote above.